hq/test/logic/IamRemediationTest.scala

```diff
@@ -98,6 +98,26 @@ class IamRemediationTest extends FreeSpec with Matchers with OptionValues with A
     }
   }
 
+  "identifyUsersWithPasswordNoMFA" - {
+    "given a user with a password and No MFA, should return that user" in {
```

Comment:
minor: could this be `given a human user with a password and No MFA, should return that user` for consistency with `given a machine user...` below?